webpack.config.js

```diff
@@ -60,14 +60,19 @@ module.exports = (env = {}) => { // pass command line arguments like `webpack ..
         plugins: [
             new webpack.NoEmitOnErrorsPlugin(),
 
+            /*** # 5/9/2017                                                                     ***/
```

Comment:
I think we really don't even need Uglify for our application since our code is so minimal compared to the rest of electron.